### ISSUE TITLE
Works when apps are full sceen. e.g. games

### ISSUE THREAD

**Issue by Saim20** [taviso/hotcorner#27]
Games are unplayable when this is active. Whenever the mouse pointer (although invisible) touches (and stays in) the corner (for the set delay), task view opens.

**Comment by taviso** [comment]
Yes - you can temporarily disable it with Ctrl+Alt+C, then press that again to turn it back on.

I guess we could detect that an application is fullscreen...

**Comment by dyskos** [comment]
This pull request will solve this issue if it's merged.
https://github.com/taviso/hotcorner/pull/29
